beacon-chain/state/types/types.go

```diff
@@ -25,6 +25,11 @@ const (
 	CompressedArray
 )
 
+type FieldIdx interface {
```

Comment:
Temporary interface so that types from both `types` and `v0types` packages can be used by the `fieldtrie` package.